src/state/trajectory/types.ts

```diff
@@ -92,3 +106,18 @@ export interface TimeUnits {
     magnitude: number;
     name: string;
 }
+
+export interface HealthCheckTimeout {
+    [requestId: string]: NodeJS.Timeout;
+}
+
+type CONVERSION_NO_SERVER = typeof CONVERSION_NO_SERVER;
+type CONVERSION_SERVER_LIVE = typeof CONVERSION_SERVER_LIVE;
+type CONVERSION_ACTIVE = typeof CONVERSION_ACTIVE;
+type CONVERSION_COMPLETE = typeof CONVERSION_COMPLETE;
+
+export type ConversionStatus =
+    | CONVERSION_NO_SERVER
```

Comment:
Does this first line need a `|`?

Reply:
logically no but i hate the way it looks without it :(